pandas/tests/groupby/test_groupby_valuecount.py

```diff
@@ -0,0 +1,120 @@
+"""
+these are systematically testing all of the args to value_counts
+with different size combinations. This is to ensure stability of the sorting
+and proper parameter handling
+"""
+
+from itertools import product
+
+import numpy as np
+import pytest
+
+from pandas import (
+    DataFrame,
+    Grouper,
+    MultiIndex,
+)
+import pandas._testing as tm
+
+
+@pytest.fixture()
+def dataframe1():
+    # Create a multi-column, multi-row index
+    TEST_DF = pd.DataFrame({
+        "c1": ["a", "b", "c"],
+        "c2": ["x", "y", "y"]
+        }, index=[0, 1, 1])
+    return TEST_DF
+    """
+    c1 c2
+    0  a  x
+    1  b  y
+    1  c  y
+    """
```

Comment:
Can you remove these; I think we should keep the tests DRY. Having the data/results in two different places, they are likely to get out of sync.